configuration/README.md

```diff
@@ -1,44 +1,53 @@
-- [Contact Summary](contact-summary.md)
+# Additional Features
+Here are some guides on additional features that can be used within your applications: 
+- [Advanced form features](forms.md)
+- [Optional settings](https://github.com/medic/medic-docs/blob/master/configuration/app-settings.md#optional-settings)
 - [Purging](purging.md)
-- [Setting up XForms in V2 (MM App)](setting-up-xforms-for-mm-app.md)
-
-3. Configuring the Webapp
-
-- [App Settings](app-settings.md)
-- [Forms](forms.md)
-- [App Settings Validations](app-settings-validations.md)
+- [Replication](replication.md)
 - [Sentinel transitions](transitions.md)
 
-4. Medic Gateway
-
+# Other components
+## Medic Gateway
 - [Medic Mobile Gateway Configuration](gateway-config.md)
 - [List of Phones that Work Well with Medic Gateway](gateway-phones.md)
+- 
 
-5. Medic-Os
-- [Install SSL Cerificates to VM](install-ssl-certificates-to-vm.md)
-
-6. Users
-- [Pyxform](pyxform.md)
-- [Supervisor creation](supervisor-creation.md)
-
-7. Data
-- [Data deletion](data-deletion.md)
-- [Revalidate Invalidated Reports](revalidate-invalidated-report.md)
+## Medic Collect
+- [Creating XForms for Medic Collect](create-xforms-for-medic-collect.md)
+- [Updating Collect forms over the air](collect-form-update-over-the-air.md)
 
+# Support Guides
+- [CouchDB Authentication](couchdb-authentication.md)
+- [Connecting to RDBMS in Windows](connecting-to-rdbms-in-windows.md)
+- [Importing data from CSVs](csv-to-docs.md)
+- [Accessing DB of production instance](direct-access.md)
+- [Configuring the automated short ID](shortcode-identifiers.md)
+- [Installing the Standard configuration](installing-a-standard-project.md)
+- [Updating the Standard forms](update-standard-forms.md)
+
+# Legacy Documentation
+- [Installing TRB files to Turbo Sim on Windows](legacy/installing-trb-windows.md) → No longer deployed
+- [Targets](legacy/targets.md) → Replaced by declarative Targets section in [this guide](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#targets)
+- [Tasks](legacy/tasks.md) → Replaced by declarative Tasks section in [this guide](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#tasks)
+- [Contact Summary](legacy/contact-summary.md) → Replaced by declarative Contacts section in [this guide](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#contacts)
+- [Setting up XForms in V2 (MM App)](legacy/setting-up-xforms-for-mm-app.md) → Replaced by Forms section in [this guide](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md#forms)
+- [Install SSL Cerificates to VM](legacy/install-ssl-certificates-to-vm.md) → Not supported for Medic projects. Self hosting instructions to come in CHT
+- [Pyxform](legacy/pyxform.md) → Should now use [medic-conf](https://github.com/medic/medic-conf#installation)
```

Comment:
medic-conf still uses pyxform.

Reply:
The medic-conf installation guide includes the required version of pyxform